sky/provision/kubernetes/utils.py

```diff
@@ -1059,7 +1068,48 @@ def get_endpoint_debug_message() -> str:
                                           debug_cmd=debug_cmd)
 
 
-def combine_pod_config_fields(config_yaml_path: str) -> None:
+def merge_dicts(source: Dict[Any, Any], destination: Dict[Any, Any]):
```

Comment:
make this function private? Seems only used within the same file?

Reply:
It's also used in authentication.py, where we use merge_dicts when setting up the secrets..